pkg/auth/providers/genericoidc/genericoidc_provider.go

```diff
@@ -131,12 +131,18 @@ func (g *GenOIDCProvider) groupToPrincipal(groupName string) v3.Principal {
 func (g *GenOIDCProvider) getRedirectURL(config map[string]interface{}) string {
 	authURL, _ := baseoidc.FetchAuthURL(config)
 
-	return fmt.Sprintf(
+	redirectURL := fmt.Sprintf(
 		"%s?client_id=%s&response_type=code&redirect_uri=%s",
 		authURL,
 		config["clientId"],
 		config["rancherUrl"],
 	)
+
+	if config["acrValue"] != nil {
+		redirectURL += fmt.Sprintf("&acr_values=%s", config["acrValue"])
+	}
+
+	return redirectURL
```

Comment:
Not really part of the PR but are we ok about silently skipping/ignoring the error from FetchAuthURL? The redirect URL will be incomplete in that case.

Reply:
Since this is part of the shared code (the original keycloak oidc implementation also uses this), it's expected that we won't always have a configured ACR value.

Reply:
I was referring to the line before with `authURL, _ := baseoidc.FetchAuthURL(config)`, but because it was not part of the code changes, I was not able to select it.

I don't think it's a big issue, by the way. However, suppressing errors without returning or logging them catches my attention.